CONTRIBUTING.md

```diff
@@ -627,6 +631,74 @@ required.
 Essentially, the steps you have to take to deploy manually are the same that
 `skaffold` will perform, and they're outlined at the previous section.
 
+#### Kubernetes E2E tests
+
+Kubernetes integration is quite complicated, a lot of things can go wrong.
```

Comment:
```suggestion
Kubernetes integration has a lot of parts that can go wrong.
```

Slight reword so we don't imply that kubernetes is too complicated for the user.